CHANGELOG.md

```diff
@@ -3,14 +3,13 @@
 ## 1.13.0 (TBD)
 
 ### New Features
+
+- Use `split_blocks=True` by default during `to_pandas` conversion for optimal memory allocation.
```

Comment:
Better tell people what this parameter is about. Most users don't know what it is.